blog/2023-05-18-how-to-use-conda-build.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: "How to use conda-build to build a python package with C/C++ dependencies"
+slug: "2023-05-18-how-to-use-conda-build.md"
+authors: [vasvi]
+tags: [conda]
+image: img/blog/2023-02-01-matrix/conda-community-space-matrix.png
+description: "We are pleased to announce that the conda community has migrated our chat rooms to the Matrix/Element platform."
```

Comment:
I clearly need to get better at reading metadata. The image and description look like they were copy pasted and never changed.

Reply:
I wanted to copy the template, but I think I forgot to change the description. I'll edit it with a new commit.